src/test/java/io/outfoxx/swiftpoet/test/FileSpecTests.kt

```diff
@@ -109,7 +149,7 @@ class FileSpecTests {
 
             extension Observable {
             
-              func test() -> Element {
+              func test() -> RxSwift.Observable.Element {
```

Comment:
This is the only side-effect of this change, but not sure if its bad? The test still passes which was added as part of https://github.com/outfoxx/swiftpoet/pull/55

Reply:
Why do you need this change?

Reply:
This is what I am also trying to figure out. This is caused by my change.

Reply:
Seems less ideal; but, it's not awful

Reply:
I would love if we can fix this case and gain both the fix and the short-hand imports in these cases. I think it requires some more changes to ExternaTypeSpec to make ths happen...

Reply:
I understand now. And I don't think it matters since it's generated code?

Reply:
I think its OK. I am considering running with this change for a release given that no other test fails and I have tested it locally with Wire...